private-templates-service/client/src/components/AdaptiveCardPanel/AdaptiveCardPanel.tsx

```diff
@@ -47,14 +50,16 @@ class AdaptiveCardPanel extends React.Component<Props> {
           </TemplateNameAndDateWrapper>
           <TemplateStateWrapper style={{ justifyContent: "center" }}>
             <StatusIndicator
-              state={
-                template.isLive
-                  ? PostedTemplate.StateEnum.Live
-                  : PostedTemplate.StateEnum.Draft
+
+              state={template.instances && template.instances[0] && template.instances[0].state
+                ? template.instances[0].state
+                //  should never reach else statement
+                : PostedTemplate.StateEnum.Draft
```

Comment:
We are looking at the state of the `[0]`th element but we pass the `version` into `templateVersion`, is that correct?

See line 42 `<AdaptiveCard cardtemplate={template} templateVersion={version} />`

Reply:
I am thinking does `version` above always correspond to the item in the 0th index

Reply:
From what I understood of what Grace told me, the template versions are stored in the `Template` object ordered from most recent (0th index) to least recent (last index). Since this is the Adaptive Card panel, we are always displaying the most recent version of the template. This is why the `state` is based off the template at the 0th index.

`version` is the actual string indicating the numeric value of the most recent template.